.github/workflows/build.yml

```diff
@@ -526,6 +526,21 @@ jobs:
         - run: cargo install taplo-cli --locked
         - run: taplo format --check
 
+  typos:
+    name: Typos
+    runs-on: ubuntu-latest
+    steps:
+      - name: Checkout sources
+        uses: actions/checkout@v4
+        with:
+          persist-credentials: false
+
+      - name: Install rust toolchain
+        uses: dtolnay/rust-toolchain@stable
+      - run: cargo install typos-cli --locked
+      - run: typos .*.toml
```

Comment:
need NOTE or TODO comment for this - I cannot understand why this additional step is needed to check `.*.toml` files